### PR TITLE
Try out complexity checking

### PR DESCRIPTION
**Pull request by salbertson**

### CODE REVIEW

**Comment by salbertson** [comment]
@gylaz, here's the other PR.

File: .ruby-style.yml [hunk 1]
```diff
@@ -504,15 +504,15 @@ Metrics/LineLength:
...
 Metrics/MethodLength:
-  Description: Avoid methods longer than 10 lines of code.
+  Description: Avoid methods longer than 20 lines of code.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#short-methods
-  Enabled: false
-  CountComments: false
-  Max: 10
+  Enabled: true
+  CountComments: true
+  Max: 20
```
Comment (croaky) [inline .ruby-style.yml]: Don't want to get aggressive and shoot for 5?

https://robots.thoughtbot.com/sandi-metz-rules-for-developers

Comment (salbertson) [inline .ruby-style.yml]: No, not really. How about the default of 10?

File: .ruby-style.yml [hunk 2]
```diff
@@ -504,15 +504,15 @@ Metrics/LineLength:
   - http
   - https
 Metrics/MethodLength:
-  Description: Avoid methods longer than 10 lines of code.
+  Description: Avoid methods longer than 20 lines of code.
```
Comment (Koronen) [inline .ruby-style.yml]: Update this line to say `10` too?

Comment (salbertson) [inline .ruby-style.yml]: Yes, thank you. Done.

**Comment by salbertson** [comment]
@gylaz thoughts?

**Comment by gylaz** [comment]
:+1:

**Pull request merged**